pe.xml

```diff
@@ -15,6 +15,33 @@
     <tax id="53" name="Impuesto General a la Ventas" rate="16.000"/>
     <tax id="54" name="Impuesto de Promocion Municipal" rate="2.000"/>
   </taxes>
+	<states>
+		<state name="Amazonas" iso_code="AMA" country="PE" zone="South America" />
+		<state name="Ancash" iso_code="ANC" country="PE" zone="South America" />
+		<state name="Apurímac" iso_code="APU" country="PE" zone="South America" />
+		<state name="Arequipa" iso_code="ARE" country="PE" zone="South America" />
+		<state name="Ayacucho" iso_code="AYA" country="PE" zone="South America" />
+		<state name="Cajamarca" iso_code="CAJ" country="PE" zone="South America" />
+		<state name="Callao" iso_code="CAL" country="PE" zone="South America" />
```

Comment:
```suggestion
		<state name="El Callao" iso_code="CAL" country="PE" zone="South America" />
```

Reply:
Thank you for reviewing my PR. I updated correction "Callao" to "El Callao" (as it is stated in the ISO.org website).

A further investigation, the official country website (https://www.peru.gob.pe/directorio/pep_directorio_poderes.asp?cod_poder=7) shows "Callao" without any pronouns ("El"), but it preserves the pronoun "La" in "La Libertad".

So I went to Perú's Constitution (https://www.congreso.gob.pe/Docs/constitucion/constitucion/Constitucion-politica-Peru.pdf) in its "duodécima" article, which states

> Duodécima. La organización política departamental de la República comprende los departamentos siguientes:
Amazonas, Ancash, Apurímac, Arequipa, Ayacucho, Cajamarca, Cusco, Huancavelica, Huánuco, Ica, Junín, La Libertad, Lambayeque, Lima, Loreto, Madre de Dios, Moquegua, Pasco, Piura, Puno, San Martín, Tacna, Tumbes, Ucayali; y la Provincia Constitucional del Callao.

So all names matches the ones at ISO website, except for "El Callao" which is called "Provincia Constitucional del Callao".

So it doesn´t seems the government itself or its population uses the formal and larger name. I believe if we stick with the less proper name "Callao" or "El Callao" with its pronoun, we will be fine.

PS. I noticed after I commited your suggested change, "El Callao" is not in proper line following the alphabetical order. I wonder if I should force update the patch. Although not that it matters, it doesn´t affect the final result (PS will order them alphabetically when showing them to the user), it is just a personal thing on the way I read the xml.

Reply:
Hi @rickygzz, thanks for this explanation. @eternoendless is a Spanish native speaker, so maybe he can help you decide whether we should use "El Callao" or "Callao".

Reply:
I don't know what the custom in Peru is for this region, but I would personally not use the pronoun. Since the capital city is called "Callao" I think it's a safe bet to call the province just Callao as well.